.doc_gen/metadata/dynamodb_metadata.yaml

```diff
@@ -1627,6 +1700,23 @@ dynamodb_Scenario_GettingStartedMovies:
                 questions at a command prompt.
               snippet_tags:
                 - ruby.example_code.dynamodb.helper.Question
+    Swift:
+      versions:
+        - sdk_version: 1
+          github: swift/example_code/ddb/basics

```

Comment:
This should be swift/example_code/ddb -- point to the service folder, not the scenario.